gradle/libs.versions.toml

```diff
@@ -15,6 +16,7 @@ kotlinPoet-ksp = { module = "com.squareup:kotlinpoet-ksp", version.ref = "kotlin
 ksp = { module = "com.google.devtools.ksp:symbol-processing-api", version.ref = "kspVersion" }
 kotlinCompileTesting = { module = "com.github.tschuchortdev:kotlin-compile-testing", version.ref = "kotlinCompileTesting" }
 kotlinCompileTestingKsp = { module = "com.github.tschuchortdev:kotlin-compile-testing-ksp", version.ref = "kotlinCompileTesting" }
+kotest-assertions-core = { module = "io.kotest:kotest-assertions-core", version.ref = "kotest" }
 assertj = { module = "org.assertj:assertj-core", version.ref = "assertj" }
```

Comment:
Do you think we can somehow use only Kotest and remove AssertJ? 🤔 

(Not for this PR, of course)

Reply:
Oh, yeah. I missed AssertJ there 🙈 I'll create a separate PR to convert to kotest (which is more kotlin friendly IMHO hehe)